### PR TITLE
Move promote/resources.md.erb to promote/resources.md.partial

### PR DESCRIPTION
**Pull request by bethanyaconnor**
This PR creates a lot of partials that simply resolve different URLs. I don't know of a better way to handle these but let me know if there is one!

### CODE REVIEW

**Comment by breville** [comment]
Would it be interesting to have a centralised list of URLs and tags in a single file?

We could write something specific, or we could perhaps parameterise an erb file?

**Review by Hamms — APPROVED** [review]
I think for now, this is the preferred way to handle this; we could also consider creating a single directory for all templates that just represent URLs.

**Comment by bethanyaconnor** [comment]
It would be good to clean these up, even if it's just putting all the URL partials in a folder, but probably in a future PR (after we've done the rest of the conversions?).

File: pegasus/sites.v3/hourofcode.com/public/promote/resources.md.partial [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 
 # Promote the Hour of Code
-### Find all the resources you need to bring attention to your Hour of Code. Not sure where to begin? Start with our <a href="<%= resolve_url('/how-to') %>">how-to guide for hosting an Hour of Code</a>! 
+### Find all the resources you need to bring attention to your Hour of Code. Not sure where to begin? Start with our [how-to guide for hosting an Hour of Code]({{ how_to_url }})!
 
 ***
 
-<%= view :promote_handouts %>
-<%= view :promote_videos %>
+{{ view :promote_handouts }}
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/promote/resources.md.partial:16]: Actually these ones that use `view :` don't seem to work...

**Pull request merged**